tunnel/conncheck.go

```diff
@@ -42,7 +42,7 @@ func (s API) CheckTunnel(ctx context.Context, req CheckTunnelRequest) (*CheckTun
 
 	if err := checkConnectivity(ctx, details.ConnectionDetails); err != nil {
 		s.Stats.Incr("status_check", stats.Tags{"success": false}, 1)
-		return &CheckTunnelResponse{Success: false, Error: err.Error()}, nil
```

Comment:
@andrewpage We should probably not reveal the connection error to end user. If any thing failed to connect, we will just expose the IP and port that is failing, which is sensitive cause there is no protection and anyone can connect to it using our worker in the same NAT? 

Instead we should probably just return the error we persisted in db. This error usually indicates some useful information, like DNS error when talking to bastion server, or ssh error when trying to connect with improper private key.